src/store.js

```diff
@@ -573,8 +573,18 @@ function EvolvStore(options) {
     let removeConfig = [];
     let removeValue = [];
     outstandingValuePromises.concat(outstandingConfigPromises).forEach(function(promise) {
-      if (promise.source === GENOME_SOURCE && (!promise.key || !expKeyStatesHas(genomeKeyStates, 'loaded', promise.key))) {
-        return;
+      if (promise.source === GENOME_SOURCE) {
+        if (!promise.key) {
+          return;
+        }
+
+        if (version === 1) {
+          if (genomeKeyStates.experiments.size <= 0) {
+            return;
+          }
```

Comment:
It was resolving keys it had received. Now it's resolving the keys it has received, and if it didn't receive a requested key, it will resolve it as undefined once the genome has loaded.